### PR TITLE
Fixing bnc#967178 & bnc#965311

### PR DESCRIPTION
**Pull request by varkoly**
- bnc#967178 Edit password policy for ldap authentication server in yast returns "undefined method" error
- bnc#965311 yast ldap-server-ssl-check does not handle ldaps only ldap+starttls works

### CODE REVIEW

File: package/yast2-auth-server.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Mar 29 10:01:57 UTC 2016 - varkoly@suse.com
+
+- bnc#967178 Edit password policy for ldap authentication server in yast returns "undefined method" error
```
Comment (kobliha) [inline package/yast2-auth-server.changes]: Peter, we usually wrap at 80 characters. Please, see the rest of the file.

**Comment by HouzuoGuo** [comment]
Any chance to get it merged?

**Comment by jreidinger** [comment]
well, it have to be in mergable state. Also travis should not fail. Then it is on you @HouzuoGuo to review it and merge ;)

**Comment by HouzuoGuo** [comment]
looks like the commits were merged a long time ago, this pr is made by a mistake.

**Pull request closed without merging**